apps/site/next.dynamic.constants.mjs

```diff
@@ -27,6 +28,19 @@ export const BLOG_DYNAMIC_ROUTES = [
     .flat(),
 ];
 
+/**
+ * This constant is used to create static routes on-the-fly that do not have a file-system
+ * counterpart route. This is useful for providing routes with matching Layout Names
+ * but that do not have Markdown content and a matching file for the route
+ *
+ * @type {Array<string>} A Map of pathname and Layout Name
+ */
+export const ARCHIVE_DYNAMIC_ROUTES = [
+  // Creates dynamic routes for downloads archive pages for each version
+  // (e.g., /download/archive/v18.20.8, /download/archive/v20.19.2)
+  ...provideReleaseVersions().map(version => [`download/archive/${version}`]),
```

Comment:
This is incorrect. This should only provide the path variables. Based on the current path variables. This should literally only be the versions, meaning this can be:

```tsx
export const ARCHIVE_DYNAMIC_ROUTES = [
  // Creates dynamic routes for downloads archive pages for each version
  // (e.g., /download/archive/v18.20.8, /download/archive/v20.19.2)
  ...provideReleaseVersions(),
];
```